druid/src/widget/split.rs

```diff
@@ -88,38 +98,89 @@ impl<T> Split<T> {
         self.splitter_size = splitter_size;
         self
     }
+
+    /// Set the minimum width of the splitter area.
+    ///
+    /// The minimum splitter area defines the minimum width of the area
+    /// where mouse hit detection is done for the splitter.
+    /// The final area is either this or the splitter size, whichever is greater.
+    ///
+    /// This can be useful when you want to use a very narrow visual splitter,
+    /// but don't want to sacrifice user experience.
+    ///
+    /// The default minimum splitter area is `6.0`.
+    pub fn min_splitter_area(mut self, min_splitter_area: f64) -> Self {
+        assert!(
+            min_splitter_area >= 0.0,
+            "min_splitter_area must be 0.0 or greater!"
+        );
+        self.min_splitter_area = min_splitter_area;
+        self
+    }
+
     /// Set whether the splitter's split point can be changed by dragging.
     pub fn draggable(mut self, draggable: bool) -> Self {
         self.draggable = draggable;
         self
     }
+
     /// Builder-style method to set whether the splitter handle is drawn as a solid rectangle.
     ///
     /// If this is `false` (the default), it will be drawn as two parallel lines.
     pub fn fill_splitter_handle(mut self, solid: bool) -> Self {
         self.solid = solid;
         self
     }
-    fn splitter_hit_test(&self, size: Size, mouse_pos: Point) -> bool {
+
+    /// Returns the width of the area of mouse hit detection.
```

Comment:
'width' is kind of confusing for a vertical splitter, but whatever 🤷‍♂

Reply:
'breadth', so we're not implying directionality (vs 'width' or 'height')?

Reply:
I'd used 'size' before, because I hadn't been able to come up with 'breadth'

Reply:
I think breadth is probably more confusing than advantageous. I think size is good?

Reply:
Yeah the usage of `width` is a bit confusing. Previously `Split` used both `size` and `width` to refer to this and I didn't spend much time thinking about it. I think it's best to use only one word for this concept and I'll do that update.

As for what word to use, `size` is decent. I can also think of `thickness` and `cross length`. I kind of like `thickness`.

Reply:
whichever you like; I still prefer size I think, thickness has z-axis implications for me?

Reply:
Well there's no z-axis involved with `Split` so that's an unlikely clash. Also [Flutter seems to use thickness](https://api.flutter.dev/flutter/material/Divider/thickness.html).

That said, I went with `size` for now due to its shortness and better fit with `area`.